src/Components/Home/QuantidadeProdutos.js

```diff
@@ -0,0 +1,27 @@
+// import React, { Component } from 'react'
```

Comment:
Código comentado pode e deve ser retirado antes da entrega final do projeto! Lembrem de revisar e tirar o que for desnecessário.